sound/soc/intel/boards/cnl_rt700.c

```diff
@@ -105,7 +105,8 @@ static const struct snd_soc_dapm_widget cnl_rt700_widgets[] = {
 static const struct snd_soc_dapm_route cnl_rt700_map[] = {
 	/*Headphones*/
 	{ "Headphones", NULL, "HP" },
-	{ "Speaker", NULL, "SPK" },
+	{ "Speaker", NULL, "SPOL" },
+	{ "Speaker", NULL, "SPOR" },
```

Comment:
I think this should be a different route (in a different structure), e.g. we need to keep the ability to test ALC700 only for on-board solutions, without RT1308. So I'd rather have a quirk and different routes really.